test/release/examples/primers/taskParallel.chpl

```diff
@@ -97,3 +97,72 @@ coforall i in 1..n {
 // The order of output is undefined.
 writeln("5: output from main task");
 
+/*
+.. _primers-taskparallel-task-intents:
+
+Task Intents
+------------
+
+The body of a task construct name some variables declared outside,
+or "outer variables". If so, "shadow variables" are introduced.
+Each task created by the task construct gets its own set of
+shadow variables, one per outer variable.
+
+ - Each shadow variable behaves as if it were a formal argument
+   of the task function for the task. The outer variable is passed
+   to this formal argument according to the argument intent
+   associated with the shadow variable, which is called a "task intent".
+
+ - The name of an outer variable lexically in the task construct
+   implicitly refers to the corresonding shadow variable.
+
+ - Each shadow variable is deallocated at the end of its task.
+
+The default argument intent is used by default. For numeric types,
```

Comment:
I feel as though something in here ought to refer to "the variable's type" to make it clear that there isn't a single default argument intent, but to get people thinking about the per-type default intents for normal procedures.  Is there a cross-reference we could/should add?  (e.g., do we have a procedures primer that covers default intents?)

Reply:
We have a [table](https://chapel-lang.org/docs/master/language/spec/procedures.html#abstract-intents-table) of default intents.

Reply:
I added a link to the spec, and it below I distinguish between numerics, arrays, syncs, etc.